src/poker/game/GamePlay.java

```diff
@@ -129,34 +100,21 @@ public PossibleAction getAction(PossibleAction lastAction, Player activePlayer,
                 return PossibleAction.FOLD;
             }
         } else {
-            System.out.println("1 check, 2 raise");
-            decision = scanner.nextInt();
-            if (decision == 1) {
-                return PossibleAction.CHECK;
-            } else {
-                return PossibleAction.RAISE;
-            }
+          return getActionFor(" check ", " raise ", PossibleAction.CHECK, PossibleAction.RAISE);
         }
     }
-    public void preFlopDetails(Player playerBigBlind, Player playerSmallBlind) {
-        actionHandler.updatePotAndStackAmount(gameState.bigBlind, playerBigBlind);
-        actionHandler.updatePotAndStackAmount(gameState.smallBlind, playerSmallBlind);
-        playerSmallBlind.increasePlayermoneyOnStreet(gameState.smallBlind);
-        playerBigBlind.increasePlayermoneyOnStreet(gameState.bigBlind);
-        GamePlayUtils.printPreFlopDetails(playerBigBlind, playerSmallBlind, result, gameState);
+    public PossibleAction getActionFor(String stringDecision, String stringDecision2, PossibleAction possibleAction,PossibleAction possibleAction2){
```

Comment:
a na chuj stringDecision i stringDecision2 a pozniej robisz "1" + stringDecision + "2" + stringDecision2 odrazy przekaz caly ten string jako "1 limp 2 rasie" a nie rozbiles to na dwa stringi